### ISSUE TITLE
Simplifier questions

### ISSUE THREAD

**Issue by bixiou** [bixiou/oecd_climate#8]
Vu les réponses des gens, on a trop de questions ou on leur donne des options tres longues et complexes a lire. Au lieu de ca, on doit avoir une question concrete (quitte un peu longue), mais ensuite des options de réponse tres simples, du genre “d’accord/pas d’accord”, des échelles de 1 a 10 d’intensité, etc.. Actuellement, les optons sont très longues a lire et ne sont pas exhaustives et exclusives. Ca va être difficilement utilisable tel quel.

Je vous donne un exemple. Une question du genre: “Do you think humanity can stop emitting greenhouse gases and halt climate change?

o The evolution of the climate is beyond our control, it is an illusion to think humanity can harness it  (1)

o We could technically stop emissions, but this would cause more harm than climate change itself, so we’d better adapt to climate change than try to stop it  (2)

o We could and we should stop emissions, but unfortunately this is not the way we are taking  (3)

o We could stop emissions, and we will eventually succeed in doing so within the next century, thanks to awareness raising and stringent public policies  (4)

o We do not have to worry too much about climate change nor enact stringent public policies, because new technologies or habits will suffice to prevent disastrous climate change  (5)

o Don't know, don't say  (6)

Est ingérable pour les gens. Ca mixe policy action, technology, fairness issues belief in climate change.. on peut meme pas “coder” cette question pour l’utiliser comme outcome, car chaque option de réponse est un theme a part. 

Au contraire, il faudrait “to what extent do you think that humankind has the means to stop greenhouse gas emissions and halt climate change” from 1 to 10 where 1 is “not at all” to 10 “has all the means necessary.” Ou quelque chose de ce type. 
Donc ce sera ici une question sur la FEASIBILITY. 

Si on veut une question sur technologies ou habits, faut poser une autre question simple la dessus..

**Comment by bixiou** [comment]
After scanning all questions, this problem might concern questions 3.7, 7.5, 7.10.
- For 3.7 (environmental views), I've answered by email. I think it's not that complex, after all. I am open to adjustments of course but I don't see how to simplify it.
- For 7.10 (effect of climate policies on lifestyle), I think the criticism doesn't really apply.
- For 7.5 (the one explicitly mentioned above), we could split it into three questions with 1-10 scales:

  * feasibility: Is it technically feasible to stop greenhouse gas emissions and stop climate change while keeping our industrial civilization?
  * prediction: Will humanity succeed in stopping greenhouse gas emissions and halting climate change?
  * decision: To what extent should we enact stringent public policies to halt climate change?

**Comment by bixiou** [comment]
We could also split 7.10 into two 1-10 scales actually, though I am not sure it makes it clearer:
- If we decided to halt climate change, would we be able to maintain our lifestyle?
- Would policies aimed at halting climate change impose overall economic costs or rather bring opportunities for our economy?

For the record, the question was:
If we decided to halt climate change, what would be the effect on our lifestyle? The policies aimed at halting climate change ... (Multiple answers are possible)
- ... would be an opportunity for our economy: it would create jobs, reduce air pollution, and improve our lifestyle
- ... would be costly, but we would be able to maintain our lifestyle (e.g. increasing thermal insulation of housing, replacing cars running on fossil fuels by electric cars, electricity from coal by renewables or nuclear, etc.)
- ... would require a deep change in our lifestyle (e.g. reducing energy consumption, eating much less meat, driving less and using public transport or cycling instead, etc.)
- Don't know, or prefer not to say

**Comment by BPlanterose** [comment]
Q° PNR: do we want to leave option that people cannot answer. Sometimes don't have an opinion (literature for instance goes back to Bourdieu, "L'Opinion publique n'existe pas", 1973)


Q3.4 Too long
Would rephrase like this: 
Think more broadly about the purpose of the government. What do you think the role of the government should be on a scale from 1 to 5? (where 1 means you think the government should do only those things necessary to provide the most basic government functions (e.g. ensure public order, protect against foreign threats, set basic rules for a free market economy), and 5 means you think the government should take active steps in every area it can to try and improve the lives of its citizens)

Q3.7: two dimensions here what will happen of our civilizations and to what extent should we try to prevent it (cf ≠ modals: will vs. should). Also quite close from the Q7.5 (technologies and habits sufficient)
So we could split the question in 2
	a) (Slider question) On a scale from 1 to 5 how do you think our civilizations can resist to the effects of climate change? (1:We will develop so much that climate change won't be an issue anymore; 5: We will collapse regardless of our efforts)
	b) (Slider question) On a scale from 1 to 5 how important do you think it is to make our society more sustainable? (1:Not important at all, I don't think we should focus on it; 5:Very important, we should make our society as sustainable as possible)

Q7.5: Agree with Adrien, we should split it. However we might want to give more context to the question. Is it stopping GHG emissions while keeping the same standard of living or changing it? Also, for the "prediction" question people might think of different degree of policy internvention (as: it is possible to halt CC with current trend of intervention vs. it is possible with a lof of efforts).  So maybe we could proceed as follows:
	a) (Slider question): Importance of stopping GHG emissions (1: Not important al all; 5: Very important, should be biggest priority)
	b) (Slider quesiton): How on a scale from 1 to 10, how important do you think policy intervention is to succeed stopping emitting GHG ? (1: Current trends of technologies and habits are sufficient w/o policy internvention; 10: Ambitious policies are needed)

Q7.10: agree with Adrien, we could split it into effects on economy and change in our lifestyle
	a) (Slider): If we decide to haalt CC w/ ambitious policies, what would be the effects on the economy (1: Very negative effect (destruct jobs, diminish standard of living): 5: Very postive effect (create job, improve standard of living))
	b) (Slider): If we decide to haalt CC w/ ambitious policies, what would be the effects on our lifestyle?
		1:  we would maintain current lifestyle; 10: we would witness deep changes in our lifestyle
	b.bis) alternative. Do you agree that if we decide to halt CC w/ ambitious policies it would imply deep changes in our lifestyle?

Q8.3: do we want to frame the question w/ precise figures. Equal right to pollute would limit your HH at XX miles of travel by car per year for instance. Also as Laurence hinted, should we precise poor in the world/U.S.

Q8.5: Global tax should we propose an amount (or a range) for the tax?
Q8.6: same question

Block 13: should we give possible options for the limit of miles ?
Block 15: should we propose different amount for the tax?

**Comment by bixiou** [comment]
Thank you Bluebery. I agree with most of your proposals except for:

3.7 I find it inappropriate to split because to me, this question cannot be seen as two independent dimensions. Indeed, 3.7a lacks an option as both are fatalistic: "On a scale from 1 to 5 how do you think our civilizations can resist to the effects of climate change? (1:We will develop so much that climate change won't be an issue anymore; 5: We will collapse regardless of our efforts)" => I am not a techno-optimist neither am I a doomer, what should I answer? This is why the original question was a multiple choice including "We should make our society as sustainable as possible to avoid irreversible damages". Furthermore, 3.7b is already addressed by your 7.5a (or my 7.5c).

7.5 I'd split it into 3 questions, not 2. You propose a. decision and b. need for policies. I think your b. is already addressed by 3.7, and I propose a. feasibility, b. prediction, c. decision. Prediction is here to give a flavor of what people think will happen, so in my view we do not need to specify the scenario (policy intervention or not). Also, I prefer my formulation for "decision": "should we..." rather than "is it important...".

8.3 The issue is that the limit is decreasing every year (not constant) and not binding (as permits are tradable). Also, here the question is purposely asked from an abstract fairness point of view. So figures do not seem necessary to me here.

8.5 Here we can specify numbers, but then we'd need to explain the proposal in more detail (i.e. say it would eradicate extreme poverty and halt climate change). I am OK for both the current, short wording, or the detailed, long one.
8.6 Same as above: we can indeed specify a precise wealth tax for example (e.g. a 0.5% tax on wealth between 1 and 10M$, 1% between 10 and 50M$, 3% between 50M and 1G$ and 6% above 1G$), there is a trade-off between precision and brevity. 

13.You mean the limit in CO2 per mile? We can re-use what is in the video, yes. Though I am not sure this extra information is worth the extra length.

15.What do you mean different amounts? Specifying a price trajectory? We could, indeed. We'd have to change the video in such case.

Also, I think Bluebery's idea to go for a scale from 1 to 5 is a good one. Indeed, an odd number of responses allows respondents to answer "in the middle", and higher number of responses puts a risk that respondents don't know how to respond (e.g. "should I go for 7 or 8 over 10?) and do not share a common language.

**Comment by BPlanterose** [comment]
Ok so maybe for 3.7 we could rephrase like this:
"On a scale from 1 to 5, would you say that strong policy intervention is needed to avoid irreversible damages due to climate change on our civilization?"
We could also have follow-up questions depending on the answers to 3.7. For instance if below 3, then ask a question on doomer vs. techno-optimist.

For 13 and 15, yes I was thinking of giving figures like price trajectories to make the proposition less abstract but if it implies changing the video it might be too much of a hassle for not so much benefits.

**Comment by bixiou** [comment]
For 13 we wouldn't have to change the video, only for 15 (but not sure this extra info is really important). 

For 3.7: why not, though I don't see the point asking what you propose as it seems redundant to what we ask in 7.5. I would still prefer the original formulation.
